lib/Backend/BackwardPass.cpp

```diff
@@ -252,6 +252,84 @@ BackwardPass::CleanupBackwardPassInfoInFlowGraph()
     }
     NEXT_BLOCK_IN_FUNC_DEAD_OR_ALIVE;
 }
+
+/*
+*   We Insert ArgIns at the start of the function for all the formals.
+*   Unused formals will be deadstored during the deadstore pass.
+*   We need ArgIns only for the outermost function(inliner).
+*/
+void
+BackwardPass::InsertArgInsForFormals()
+{
+    if (func->IsStackArgsEnabled() && !func->GetJnFunction()->GetHasImplicitArgIns())
+    {
+        IR::Instr * insertAfterInstr = func->m_headInstr->m_next;
+        AssertMsg(insertAfterInstr->IsLabelInstr(), "First Instr of the first block should always have a label");
+
+        Js::ArgSlot paramsCount = insertAfterInstr->m_func->GetJnFunction()->GetInParamsCount() - 1;
+        IR::Instr *     argIninstr = nullptr;
```

Comment:
Done.

---
In reply to: [67976968](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67976968) [](ancestors = 67976968)